lib/doorkeeper/oauth/pre_authorization.rb

```diff
@@ -17,7 +18,7 @@ class PreAuthorization
       attr_reader :server, :client_id, :client, :redirect_uri, :response_type, :state,
                   :code_challenge, :code_challenge_method, :missing_param
 
-      def initialize(server, attrs = {})
+      def initialize(server, attrs = {}, resource_owner = nil)
```

Comment:
It's OK for me just because `attrs` not options, so we can add one more argument